pages/api/auth/[...nextauth].ts

```diff
@@ -39,7 +39,7 @@ export const authOptions = {
             clientSecret: process.env.FACEBOOK_CLIENT_SECRET || "",
         }),
     ],
-    adapter: PrismaAdapter(prisma),
+    adapter: PrismaAdapter(prisma) as any,
```

Comment:
Something is broken is Prisma types, will be fixed with upcoming TS / Prisma versions.